notebooks/writing_a_trace_transform_cpu_offloading.ipynb

```diff
@@ -0,0 +1,890 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # print_memory_usage_and_benchmark("thunder")
</pre><p>Commented to avoid running this on CI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Lightning-AI/lightning-thunder/pull/966/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>